ConsistencyManager/ConsistencyManager.swift

```diff
@@ -140,23 +140,23 @@ open class ConsistencyManager {
      Note that calling this method on a paused listener will not unpause it.
      - parameter listener: The consistency manager listener that is listening to a model
     */
-    open func listenForUpdates(_ listener: ConsistencyManagerListener) {
+    open func addListener(_ listener: ConsistencyManagerListener) {
         let model = listener.currentModel()
         if let model = model {
-            listenForUpdates(listener, onModel: model)
+            addListener(listener, to: model)
         }
         // Else they are listening to nothing. Let's not remove them though, since we are on a different thread, so timing issues could cause bugs.
     }
 
     /**
-     Call this method if you want to listen to a specific model. Usually, this is unnecssary and you should just use listenForUpdates(listener).
+     Call this method if you want to listen to a specific model. Usually, this is unnecssary and you should just use addListener(listener).
      This is necessary if you manually update a model and change only part of it.
      Note that calling this method on a paused listener will not unpause it.
      For a performance optimization, you may only want to add yourself as a listener for this new change (and not the whole model again).
      - parameter listener: the consistency manager
-     - parameter onModel: the model you want to listen to with this listener
+     - parameter to: the model you want to listen to with this listener
```

Comment:
I think that the name of the parameter should be `model`. For example, in the docs for `index(of element: Element)` in an Array, the parameter name is `element`.